static/js/controllers/contacts-edit.js

```diff
@@ -34,7 +34,7 @@ var _ = require('underscore');
         return UserDistrict()
           .then(function(facilityId) {
             if (!facilityId) {
-              return $q.resolve();
+              return $q.reject('Facility ID not supplied');
```

Comment:
> 

note, that if the user is not admin, and doesn't have a facility_id, then UserDistrict will promise-reject.

That's not what I'm seeing, so perhaps there is another problem...